xbmc/Application.cpp

```diff
@@ -2734,11 +2711,42 @@ void CApplication::OnApplicationMessage(ThreadMessage* pMsg)
   break;
 
   case TMSG_LOADPROFILE:
-  {
     CGUIWindowLoginScreen::LoadProfile(pMsg->param1);
     break;
+
+  default:
+    CLog::Log(LOGERROR, "%s: Unhandled threadmessage sent, %u", __FUNCTION__, pMsg->dwMessage);
```

Comment:
if it's always an error and something that must be fixed if we end up here, ok. otherwise, imo errors should be reserved for those, not things that *might* be an error